### PR TITLE
Document updates for "DR10.1"

### PR DESCRIPTION
**Pull request by geordie666**
This PR documents the updates made to fix the sub-blob bug (i.e. for "DR10.1"): 

- The "sub-blobs" bug description on the known issues page has been expanded to describe how that bug was fixed.
- The sweeps section of the files page has been updated to describe the DR10 and DR10.1 versions of the sweeps.
- The external-match files section has been updated to describe the DR10 and DR10.1 versions of the files.

### CODE REVIEW

**Comment by geordie666** [comment]
This PR addresses issue #173.

**Comment by geordie666** [comment]
@rongpu would like a new version of the webpages served as soon as possible to support the submission of his photo-z paper early next week.

**Comment by weaverba137** [comment]
I think we can do that on Monday.

File: CHANGES.md [hunk 1]
```diff
@@ -9,6 +9,9 @@
 
 ## 10.0.4 (DR10, planned)
```
Comment (weaverba137) [inline CHANGES.md:10]: Let's tag this as 10.1.0.

Comment (weaverba137) [reply CHANGES.md:10]: Unless there are other significant DR10.1-related changes planned?

File: pages/dr10/files.rst [hunk 2]
```diff
@@ -523,8 +523,15 @@ match with any Legacy Surveys ``BRICK_PRIMARY`` photometric object.
 .. _`catalogs`: ../catalogs
 .. _`Catalogs`: ../catalogs
 
-survey-dr10-south-specObj-dr16.fits
------------------------------------
+survey-dr(10|10.1)-south-specObj-dr16.fits
+------------------------------------------
+
+.. note::
```
Comment (weaverba137) [inline pages/dr10/files.rst:529]: It seems like you could just make one note that covers all of the survey files in a higher-level section.

File: pages/dr10/files.rst [hunk 3]
```diff
@@ -631,8 +673,14 @@ Users interested in database access to the Tractor `catalogs`_ can use the `NOIR
 Sweep Catalogs (``south/sweep/*``)
 ==================================
 
-10.0/sweep-<brickmin>-<brickmax>.fits
--------------------------------------
+10.[0-1]/sweep-<brickmin>-<brickmax>.fits
+-----------------------------------------
+
+.. note::
```
Comment (weaverba137) [inline pages/dr10/files.rst:679]: Similarly to the comment above, you could put this note only once immediately below the 'Sweep Catalogs' section header.

**Review by weaverba137 — CHANGES_REQUESTED** [review]
This looks OK, but I suggest moving some notes around to remove redundancy.

The branch compiles OK, so no concerns there.

File: CHANGES.md (hunk 1 above)
Comment (geordie666) [reply CHANGES.md]: Agreed. I've updated the `CHANGES.md` file accordingly.

File: pages/dr10/files.rst (hunk 2 above)
Comment (geordie666) [reply pages/dr10/files.rst]: Thanks for this suggestion. I vacillated on whether or not to include a note with every file or just one per section, so I appreciated the second opinion.

Comment (geordie666) [reply pages/dr10/files.rst]: (and I've changed to one-note-per-file-_type_).

File: pages/dr10/files.rst (hunk 3 above)
Comment (geordie666) [reply pages/dr10/files.rst:646]: Again, I appreciated this opinion and have made your suggested update.

File: CHANGES.md (hunk 1 above)
Comment (geordie666) [reply CHANGES.md]: Note that there may be other DR10 changes (John's SGA changes spring to mind). But there shouldn't be any other specific changes for DR10.1.

**Review by weaverba137 — APPROVED** [review]

**Comment by weaverba137** [comment]
OK to merge when ready.  Be sure to close #174 since I don't think this PR is set up to do that automatically.

**Pull request merged**